send-transaction-service/src/send_transaction_service.rs

```diff
@@ -236,8 +380,14 @@ impl SendTransactionService {
             }
             if let Some((nonce_pubkey, durable_nonce)) = transaction_info.durable_nonce_info {
                 let nonce_account = working_bank.get_account(&nonce_pubkey).unwrap_or_default();
+                let now = Instant::now();
```

Comment:
With decoupled retry and receive -- this change is no longer necessary as it has implicitly expired

Reply:
We still need this -- in case it is sent in the receive side and right after that dropped here.